numba/byteflow.py

```diff
@@ -974,6 +1095,29 @@ def outgoing_phis(self):
         """
         return self._outgoing_phis
 
+    @property
+    def blockstack_initial(self):
+        """A copy of the initial state of the blockstack
```

Comment:
Is this a copy or a reference?

Reply:
A copy because it was copied at the constructor.